sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/TracerProvider.java

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation;
+
+import com.azure.core.util.Context;
+import com.azure.core.util.FluxUtil;
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.core.util.tracing.Tracer;
+import com.azure.cosmos.CosmosClientException;
+import com.azure.cosmos.models.CosmosAsyncItemResponse;
+import com.azure.cosmos.models.CosmosResponse;
+import reactor.core.publisher.Mono;
+import reactor.core.publisher.Signal;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Function;
+
+import static com.azure.core.util.tracing.Tracer.AZ_TRACING_NAMESPACE_KEY;
+
+public class TracerProvider {
+    private final ClientLogger logger = new ClientLogger(TracerProvider.class);
+    private final List<Tracer> tracers = new ArrayList<>();
+    public final static String DB_TYPE_VALUE = "cosmosdb";
```

Comment:
shouldn't this be "cosmos" instead of "cosmosdb"

Reply:
@trask  ?

Reply:
both are good from my perspective 👍

Reply:
The brand of our service, our package name, etc, is cosmos. So this should match that.

Reply:
For consistency on SDK lets use "Cosmos"